.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+### Checklist
+
+Please:
+- [ ] Make sure this PR is against "dev", not "main".
+- [ ] Request a review from one of the current epiprocess main reviewers:
+      brookslogan, nmdefries.
+- [ ] Describe changes made in NEWS.md or under the "Changelog" heading, making
+      sure breaking changes (backwards-incompatible changes to the documented
+      interface) are noted. The right location for this may be in flux (see
+      [here](https://github.com/cmu-delphi/epiprocess/pull/398)).
+
+### Changelog (if not put in NEWS.md)
```

Comment:
```suggestion
- [ ] Describe changes made in NEWS.md, making
      sure breaking changes (backwards-incompatible changes to the documented
      interface) are noted. The right location for this may be in flux (see
      [here](https://github.com/cmu-delphi/epiprocess/pull/398)).

### Change explanations for reviewer
```